.github/workflows/release.yml

```diff
@@ -0,0 +1,164 @@
+name: release
+on:
+  push:
+    tags:
+      - 'u*'
```

Comment:
switch this to "v" before merge as per PR description?

Reply:
Yes, unless we want to get a full singing-and-dancing, syncing-and-releasing thing merged together and tested before we flip that...

Reply:
Done